#Exception handling

try:
	a=8
	b=0
	print(a/b)
except Exception as e:
	print('enter valid no.0', e)